autoPlay.user.js

```diff
@@ -27,8 +27,15 @@ var removeFlinching = getPreferenceBoolean("removeFlinching", true);
 var removeCritText = getPreferenceBoolean("removeCritText", false);
 var removeAllText = getPreferenceBoolean("removeAllText", false);
 
+if (typeof GM_info !== "undefined") {
+	var enableAutoRefresh = getPreferenceBoolean("enableAutoRefresh", true); // auto-refresh for GM/TM users
+} else {
+	var enableAutoRefresh = getPreferenceBoolean("enableAutoRefresh", false); // no auto-refresh for non-GM/TM users
+}
```

Comment:
These lines could be simplified:
`var enableAutoRefresh = getPreferenceBoolean("enableAutoRefresh", typeof GM_info !== "undefined");`